src/components/customComponents/button.js

```diff
@@ -8,7 +8,7 @@ class Button extends Component {
   render() {
     return (
       <button
-        type='button'
+        type={this.props.type || 'button'}
```

Comment:
use default props in this cases: https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values